agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4371,11 +4371,23 @@ bool slave_thread::handle_client_steering_request(Socket *sd, ieee1905_1::CmduMe
         message_com::send_cmdu(ap_manager_socket, cmdu_tx);
         return true;
     } else {
+
         //TODO Add handling of steering opportunity
```

Comment:
Please remove TODOs when the work is TODONE :slightly_smiling_face:

Reply:
This flow is already tested as part of the certification tests.
The goal of this PR is to respond with the appropriate messages to the controller (to pass the certification) and not to actually implement the steering opportunity logic (which is not yet defined for prplMesh).